cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -725,7 +725,6 @@ var Toolbar = new Class({
         const diff = dd.diff(this.ui.toolbar[0], newToolbar[0]);
 
         dd.apply(this.ui.toolbar[0], diff);
```

Comment:
```suggestion
        dd.apply(this.ui.toolbar[0], diff);

```

Unrelated formatting changes added.